bpf/flows.c

```diff
@@ -61,6 +66,10 @@ static inline void update_existing_flow(flow_metrics *aggregate_flow, pkt_info *
     if (aggregate_flow->start_mono_time_ts == 0) {
         aggregate_flow->start_mono_time_ts = pkt->current_ts;
     }
+
+    if (aggregate_flow->last_seen_timestamp == 0) {
```

Comment:
This should be `if (pkt->currents_ts != 0)` - i.e., always update the timestamp with the latest value.